### PR TITLE
Fixes #13200 - Ensure that the capsule installer sets up a standart p…

### PR DESCRIPTION
**Pull request by johnpmitsch**
…ulp server

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -192,18 +192,6 @@
...
-      server_ca_cert       => $certs::params::pulp_server_ca_cert,
-    }
-
-    pulp::child::fragment{'gpg_key_proxy':
-      ssl_content => template('capsule/_pulp_child_gpg_proxy.erb'),
-    }
-
     class { '::certs::pulp_child':
       hostname => $capsule_fqdn,
       notify   => [ Class['pulp'], Class['pulp::child'] ],
```
Comment (ehelms) [inline manifests/init.pp]: Remove the second notify here as we are removing that

**Comment by ehelms** [comment]
This needs to include your new qpid_client class

**Comment by johnpmitsch** [comment]
Tested with https://github.com/Katello/puppet-certs/pull/78 on a capsule install, which ran fine and I was able to run a capsule sync

File: manifests/init.pp [hunk 2]
```diff
@@ -191,22 +192,10 @@
...
-    }
-
-    pulp::child::fragment{'gpg_key_proxy':
-      ssl_content => template('capsule/_pulp_child_gpg_proxy.erb'),
     }
 
-    class { '::certs::pulp_child':
-      hostname => $capsule_fqdn,
-      notify   => [ Class['pulp'], Class['pulp::child'] ],
+    ::pulp::apache::fragment{'gpg_key_proxy':
```
Comment (ehelms) [inline manifests/init.pp]: I don't think you need the leading `::` in this case, otherwise looks good!

**Comment by johnpmitsch** [comment]
tested with https://github.com/Katello/puppet-pulp/pull/133 and https://github.com/Katello/puppet-certs/pull/78 on a capsule install and it ran successfully

**Comment by ehelms** [comment]
Can you please update the minimum version of puppet-certs to 0.5.0 and puppet-pulp to 3.2.0 in the metadata.json?

**Comment by johnpmitsch** [comment]
@ehelms updated

**Comment by ehelms** [comment]
Thanks @johnpmitsch !

**Pull request merged**